F50/Preferences/Preferences.android.cs

```diff
@@ -16,82 +16,64 @@ public bool ContainsKey(string key)
 			{
 				using (var sharedPreferences = GetSharedPreferences())
 				{
-					if (sharedPreferences == null)
-						return false;
-
 					return sharedPreferences.Contains(key);
 				}
 			}
 		}
 
-		public bool Remove(string key)
+		public void Remove(string key)
 		{
 			lock (locker)
 			{
 				using (var sharedPreferences = GetSharedPreferences())
+				using (var editor = sharedPreferences.Edit())
 				{
-					if (sharedPreferences == null)
-						return false;
-
-					using (var editor = sharedPreferences.Edit())
-					{
-						return editor.Remove(key).Commit();
-					}
+					editor.Remove(key).Commit();

```

Comment:
Same here.. editor?.Remove(key)?.Commit(); ?

Reply:
Same here, I don't see anything that refers to Edit() returning null, so I think we're ok to just bubble up any weird exceptions that we might encounter for now.

Reply:
roger that